functions/Find-DbaOrphanedFile.ps1

```diff
@@ -0,0 +1,225 @@
+﻿
+Function Find-DbaOrphanedFile
+{
+<#
+.SYNOPSIS 
+Find-DbaOrphanedFile moves/removes orphaned database files

```

Comment:
The synopsis should state what the function does, not necessarily what you can do with the output. You might put those as examples with using `Remove-Item` or `Move-Item`. It implies that the function will do those actions, which is a bit misleading.